cpu.Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM tensorflow/tensorflow:1.8.0-py3
 RUN mkdir /root/mimic2
 COPY . /root/mimic2
 WORKDIR /root/mimic2
+RUN apt-get update
+RUN apt-get install -y python3-tk
```

Comment:
Is this needed? TK is a GUI package right?

Reply:
Running it with original Dockerfile didn't work (for me). After installing/adding the python3-tk package it did work. But this should be verified or rolled back.

Reply:
Commit rolled back.
If it's needed i'll add a seperate pr in future.